lib/roots_web/schema/cookbook_types.ex

```diff
@@ -16,19 +16,15 @@ defmodule RootsWeb.Schema.CookbookTypes do
   end
 
   object :cookbook_queries do
-    @desc "Get all cookbooks"
-    field :cookbooks, list_of(:cookbook) do
-      resolve(&Resolvers.CookbookResolver.list/3)
-    end
-
     @desc "Get specific cookbook"
     field :cookbook, :cookbook do
```

Comment:
IIRC frontend is expecting the endpoint for getting specific cookbooks to be get_cookbook, not cookbook.